.changeset/pretty-needles-accept.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/core': minor
+'@twilio-paste/react-spectrum-library': minor
```

Comment:
```suggestion
'@twilio-paste/react-spectrum-library': major
```
?

Reply:
Actually, yeah, that's fair. If its not a 1.0 any change, minor or otherwise will cause a peer dependency major ripple effect, I believe, due to the way changesets handle 0 version peer dep mismatching.

Reply:
Good call!